organize/models.py

```diff
@@ -92,6 +93,20 @@ def create_event(self):
 
         return event
 
+    def has_past_team_members(self):
+        """ For repeated events, check whether there are any common
+        team members who applied to organize again
+        """
+        previous_events = Event.objects.filter(city=self.city,
+                                               country=self.get_country_display())
+        organizers = []
+        for event in previous_events:
```

Comment:
Do we want to filter all previous events or just the last event? Imagine I have handled the first event in city X, for the second event another group of organizers got the access and the e-mail has changed, so now I don't know the password but I still was in the previous organizers.